ignite/pkg/protoanalysis/protoutil/creator.go

```diff
@@ -0,0 +1,636 @@
+package protoutil
+
+import (
+	"fmt"
+	"unicode"
+
+	"github.com/emicklei/proto"
+)
+
+// Wrap proto structs to allow easier creation, protobuf lang is small enough
+// to easily allow this.
```

Comment:
If this is a package comment doc, it should be located before `package protoutil`, and prefixed like

```go
// Package protoutil wraps proto structs to allow easier creation, protobuf lang is small enough
// to easily allow this.
package protoutil
```

Reply:
definitely intended as a package doc, will do.